src/ducks/transactions/TransactionActions.styl

```diff
@@ -20,3 +20,13 @@
 
 .TransactionActionMenuItem
     text-align left
+
+.TransactionActionButton
+    +medium-screen()
+        &:before
+            content ''
+            position absolute
+            top 0
+            bottom 0
+            width 1rem
+            right 100%
```

Comment:
can you add comment saying this is to enlarge the click area (if I've correctly understood)